src/js/main.js

```diff
@@ -53,6 +60,10 @@ var SupportKit = Marionette.Object.extend({
     },
 
     init: function(options) {
+        if (this.ready) {
```

Comment:
An important bit of code for @juliangarritano to keep in mind for app uesr login/logout